CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Changelog
 
+### 2.0.0 (unreleased)
+* Rails 5.1 support.
+* Drop Rails 3.2, 4.0 and 4.1 and Ruby 1.9 and 2.0 support.
+
 ### 0.0.12 (unreleased)
```

Comment:
When you cut a 1.0 release for the previously supported Rails versions it should also get an entry here.